resources/Makefile.erb

```diff
@@ -1,11 +1,15 @@
 <% dirnames = get_directories.map {|d| d.path } %>
-export SHELL := $(shell which bash)
+
+SHELL = $(PWD)/profiling_shell.sh
+export VANAGON_PROJECT ?= <%= @name %>
+export VANAGON_PLATFORM ?= <%= @platform.name.tr('-','.') %>
 
 tempdir := $(shell mktemp -d -p /var/tmp 2>/dev/null || mktemp -d -t 'tmp')
 workdir := $(PWD)
 
 all: file-list-before-build <%= package_name %>
 
+<%= package_name %>: export VANAGON_TARGET = create-package
```

Comment:
Beyond the individual component metrics, this was the only metric I could think of that would be useful to collect. This doesn't include downloading or unpacking sources, prepping the build host, etc, as that all happens before the makefile is executed. Is there anything I'm leaving out that we want to include?